JSIL.mscorlib/JSIL/System/NumberFormatter.Data.cs

```diff
@@ -0,0 +1,916 @@
+﻿//
+// mono/mono/metadata/number-formatter.h
```

Comment:
Please explicitly call out what license that was under in addition to the filename. Doesn't need the whole license prologue